shared/main-shared.native.js

```diff
@@ -12,10 +13,23 @@ import {getUserImageMap, loadUserImageMap} from './util/pictures'
 import {initAvatarLookup, initAvatarLoad} from './common-adapters'
 import {listenForNotifications} from './actions/notifications'
 import {persistRouteState, loadRouteState} from './actions/platform-specific.native'
-import {setRouteState, navigateUp} from './actions/route-tree'
+import {setRouteState} from './actions/route-tree'
 
-// TODO type this later
-type Props = any
+type Props = {
+  dumbFullscreen: boolean,
+  folderBadge: number,
+  menuBadgeCount: number,
+  mountPush: boolean,
+  routeDef: any,
```

Comment:
These have types in https://github.com/keybase/client/blob/bf110bbc1503c6514227fb5b860436dbe95f5dde/shared/route-tree/render-route.js#L33